pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter_test.go

```diff
@@ -2128,6 +2128,15 @@ var _ = Describe("Validating VM Admitter", func() {
 					Field:   "spec.template.spec.domain.memory.guest",
 					Message: fmt.Sprintf("Guest memory must be %s aligned", resource.NewQuantity(memory.HotplugBlockAlignmentBytes, resource.BinarySI)),
 				}),
+				Entry("guest memory with hugepages is not properly aligned", func(vm *v1.VirtualMachine) {
+					vm.Spec.Template.Spec.Domain.Memory.Guest = pointer.P(resource.MustParse("2G"))
+					vm.Spec.Template.Spec.Domain.Memory.MaxGuest = pointer.P(resource.MustParse("16Gi"))
+					vm.Spec.Template.Spec.Domain.Memory.Hugepages = &v1.Hugepages{PageSize: "1Gi"}
```

Comment:
I am probably missing something but doesn't 16%1==0?

Reply:
it's `guest' that it's not aligned, it's 2G instead of 2Gi.